### PR TITLE
feat(FEC-9180): Player's Share to social - Studio configuration

### PR DESCRIPTION
**Pull request by Yuvalke**

### CODE REVIEW

File: app/js/services/v3Properties.json [hunk 1]
```diff
@@ -462,6 +462,45 @@
...
+        "label": "Share",
+        "children": [
+          {
+            "label": "Share url",
+            "helpnote": "Share url override Current page sharing",
+            "type": "text",
+            "model": "config.plugins.share.shareUrl"
+          },
+          {
+            "label": "Embed url",
```
Comment (Yuvalke) [inline app/js/services/v3Properties.json:480]: remove and use default iframe

File: app/js/services/v3Properties.json [hunk 2]
```diff
@@ -462,6 +462,45 @@
...
+            "helpnote": "Embed url override Current page sharing",
+            "type": "text",
+            "model": "config.plugins.share.embedUrl"
+          },
+          {
+            "label": "Allow time offset",
```
Comment (Yuvalke) [inline app/js/services/v3Properties.json:486]: set true by default

File: app/js/services/v3Properties.json [hunk 3]
```diff
@@ -462,6 +462,45 @@
             "initvalue": true
           }
         ]
+      },
+      "share": {
+        "type": "featuremenu",
+        "description": "Add the share interface to the player.",
+        "model": "config.plugins.share",
+        "componentName": "playkit-share",
+        "label": "Share",
```
Comment (Yuvalke) [inline app/js/services/v3Properties.json:471]: Share and embed

File: app/js/controllers/EditCtrl.js [hunk 4]
```diff
@@ -648,7 +648,7 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
 						// save plugin name in a model
 						properties[plug].model = plug;
 						// check plugin enabled
-						if (properties[plug].enabled || $scope.playerData.config[plug] || $scope.playerData.config.plugins[plug] || ($scope.playerData.config.ui && $scope.playerData.config.ui.components && $scope.playerData.config.ui.components[plug]) || (plug === "receiver" && $scope.playerData.externals && $scope.playerData.externals[properties[plug].componentName])){
+						if (properties[plug].enabled || $scope.playerData.config[plug] || ($scope.playerData.config.plugins[plug] && !$scope.playerData.config.plugins[plug].disable) || ($scope.playerData.config.ui && $scope.playerData.config.ui.components && $scope.playerData.config.ui.components[plug]) || (plug === "receiver" && $scope.playerData.externals && $scope.playerData.externals[properties[plug].componentName])){
```
Comment (Yuvalke) [inline app/js/controllers/EditCtrl.js:651]: add config when share is active only to keep uiconf simple

File: app/js/services/v3Properties.json [hunk 5]
```diff
@@ -462,6 +462,73 @@
...
+            "type": "checkbox",
+            "model": "config.plugins.share.useNative",
+            "initvalue": false
+          },
+          {
+            "label": "Social networks",
+            "helpnote": "Define included networks.",
+            "type": "json",
+            "model": "config.plugins.share.socialNetworks",
+            "initvalue": {
```
Comment (Yuvalke) [inline app/js/services/v3Properties.json:498]: copy value from share

**Review by yairans — APPROVED** [review]

**Pull request merged**